docs/clauses/match.md

```diff
@@ -1,20 +1,20 @@
 # MATCH
 
-The `MATCH` clause allows you to specify the patterns Cypher will search for in the database. This is the primary way of getting data into the current set of bindings. It is worth reading up more on the specification of the patterns themselves in Patterns.
+The `MATCH` clause allows you to specify the patterns a query will search for in the database. This is the primary way of retrieving data for use in a query.
 
-`MATCH` is often coupled to a `WHERE` part which adds restrictions, or predicates, to the `MATCH` patterns, making them more specific. The predicates are part of the pattern description, and should not be considered a filter applied only after the matching is done. This means that `WHERE` should always be put together with the `MATCH` clause it belongs to.
+ A `WHERE` clause often follows a `MATCH` clause to add user-defined constraints to the matched patterns to manipulate the set of data returned. The constraint predicates are part of the pattern description, and should not be considered a filter applied only after the matching is done. This means that `WHERE` should always be put together with the `MATCH` clause it belongs to.

```

Comment:
Looks good! Just one thought. Don't we also use the term 'constraint' to refer to database constraints like NOT NULL, DEFAULT etc? 🤔

Reply:
Reverted~